pokemon-league-participation-manager/src/test/java/cz/muni/fi/pa165/pokemon/league/participation/manager/dao/BadgeDAOTest.java

```diff
@@ -58,9 +63,12 @@
     
     private Trainer trainerAsh;
     private Trainer trainerBrock;
+    private Trainer trainerWallace;
+    private Trainer trainerWattson;
     
     private Gym gymInBrno;
-    private Gym finalGym;
+    private Gym sootopolisGym;
+    private Gym mauvilleGym;
```

Comment:
mauvilleGym - is completely redundant here. Two gyms are enough to test everything

Reply:
While possible even if you want to have 2 badges stored in setup, you would have to issue the badge from createBadge test to a different trainer (I opted for the 3 gyms, because it keeps the challenger consistent across all badges - to me it looks neater this way).

Reply:
By the way, that just explains my reasoning for using three gyms, I'm changing it now.